src/preact10-internals.ts

```diff
@@ -1,4 +1,8 @@
 import type { Component, VNode } from 'preact';
+import { Fragment } from 'preact';
+
+// This makes enzyme debug output easier to work with by giving Fragments a name that isn't minified
+Fragment.displayName = 'Fragment';
```

Comment:
Actually, yea good call. Since this is a side-effect, I'm going to remove this from the src and put in the test/compat-shallow-renderer/utils.ts for now to avoid the side-effect altogether at the moment. I think I'll add it only if the compat shallow renderer is used so it only gets installed if the compat behavior is opted into. That way the side-effect is considered part of compat side-effects only.

I'll do that in the next PR where I integrate the shallow renderer into enzyme and determine how to only include the files if they are opted into (so side-effects like this one are opted into)